app/domain/authentication/authn_jwt/configuration_jwt_generic_vendor.rb

```diff
@@ -2,14 +2,27 @@ module Authentication
   module AuthnJwt
     # Mock JWTConfiguration class to use it to develop other part in the jwt authenticator
     class ConfigurationJWTGenericVendor < ConfigurationInterface
+      GENERIC_JWT_CONSTRAINTS = Authentication::Constraints::MultipleConstraint.new(
+            Authentication::Constraints::NotEmptyConstraint.new
+        )
+
       def self.jwt_identity(authentication_parameters)
         id_provider = Authentication::AuthnJwt::IdentityProviderFactory.new(authentication_parameters).relevant_id_provider
         id_provider.provide_jwt_identity
       end
 
       def self.validate_restrictions(authentication_parameters)
-        Authentication::AuthnJwt::ValidateResourceRestrictions.new.call(
-          authentication_parameters: authentication_parameters
+        Authentication::ResourceRestrictions::ValidateResourceRestrictions.new(
+          extract_resource_restrictions: Authentication::ResourceRestrictions::ExtractResourceRestrictions.new
```

Comment:
This is the default value for `extract_resource_restrictions` dependency.
Why do you need to override it with the same value?